seaworthy/fixtures.py

```diff
@@ -0,0 +1,42 @@
+from seaworthy.definitions import ContainerDefinition
+from seaworthy.containers.postgresql import PostgreSQLContainer
+from seaworthy.utils import output_lines
+
+
+class DjangoContainer(ContainerDefinition):
+    WAIT_PATTERNS = (r"Booting worker",)
+
+    def __init__(self, name, db_url,
+                 image="malaria"):
+        super().__init__(name, image, self.WAIT_PATTERNS, wait_timeout=20)
+        self.db_url = db_url
+
+    def base_kwargs(self):
+        return {
+            "environment": {
+                "DATABASE_URL": self.db_url,
+                "ALLOWED_HOSTS": "0.0.0.0",
+            },
+            "ports": {
+                "8000/tcp": ("127.0.0.1",),
+            }
+        }
+
+    def exec_django_admin(self, *args):
```

Comment:
Doesn't look like you're using this method?

Reply:
well remove it